src/Core/src/Platform/Windows/ViewExtensions.cs

```diff
@@ -370,14 +372,33 @@ internal static Graphics.Rect GetBoundingBox(this FrameworkElement? platformView
 
 		internal static void UnfocusControl(Control control)
 		{
-			if (control == null || !control.IsEnabled)
+			if (!control.IsEnabled)
+			{
 				return;
+			}
 
-			var isTabStop = control.IsTabStop;
-			control.IsTabStop = false;
-			control.IsEnabled = false;
-			control.IsEnabled = true;
-			control.IsTabStop = isTabStop;
+			// https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.focus?view=winrt-26100#remarks
+			// You can't remove focus from a control by calling Focused() method with Unfocused as the parameter.
+			// This value is not allowed and causes an exception. To remove focus from a control, set focus to a different control.
```

Comment:
This was the initial implementation:
https://github.com/dotnet/maui/pull/4562/files#diff-9ce2119c304c90706d054ac06d6f497b47095e9ded5b659fb32472c4cfaa866f

They had to change it to prevent a crash: https://github.com/dotnet/maui/pull/13028

My solution is crash-free and matches guidelines.